examples/README.md

```diff
@@ -7,23 +7,23 @@ Update the relevant sections under `if location == "local"` or `if location == "
 
 ### Define and initalize the Model Specification
 There are three different ways to construct a model specification:
-1. By calling the constructor of `TGISModelSpec` or `vLLMModelSpec` directly (as defined in [this file](https://github.com/fmperf-project/fmperf/blob/main/fmperf/ModelSpecs.py)).
-2. By calling `TGISModelSpec.from_yaml` or `vLLMModelSpec.from_yaml` and passing a path to a YAML file defining the specification (see example [here](https://github.com/fmperf-project/fmperf/blob/main/examples/model_specifications_tgis_one.yml) for a `TGISModelSpec`).
+1. By calling the constructor of `TGISModelSpec` or `vLLMModelSpec` directly (as defined in [this file](/fmperf/ModelSpecs.py).
+2. By calling `TGISModelSpec.from_yaml` or `vLLMModelSpec.from_yaml` and passing a path to a YAML file defining the specification (see example [here](/examples/model_specifications_tgis_one.yml) for a `TGISModelSpec`). There is a one-to-one mapping between the constructor arguments and the key-value pairs in the yaml files.
 
 ### Define and initalize the Workload Specification
```

Comment:
This section title is repeated twice

Reply:
Thanks for the review.
Not sure I follow this.